src/layouts/components/Editor/components/ImageBubbleMenu/ImageBubbleMenu.tsx

```diff
@@ -117,7 +117,7 @@ const ImageLinkButton = () => {
           )}
         </Popover>
 
-        <Popover placement="bottom">
+        {/* <Popover placement="bottom">
```

Comment:
why is this commented? is there a future todo for this?

Reply:
doesn't work - see #1721